sdk/personalizer/Azure.AI.Personalizer/Microsoft.Azure.AI.Personalizer.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
Is this project file for track 1 SDK, while `Azure.AI.Personalizer.csproj` is for track 2?
Also, this project is not included in the one solution file, so I am wondering if this is intended to be used.

Reply:
This project was autogenerated, but it is not needed so I will go ahead and remove it.

Reply:
Discussed offline and removing the csproj was recommended.